api/v1alpha1/database_webhook.go

```diff
@@ -120,29 +120,6 @@ func (r *Database) Default() {
 
 var _ webhook.Validator = &Database{}
 
-func ndbServerSpecValidatorForCreate(ndb *NDB, allErrs field.ErrorList, ndbPath *field.Path) field.ErrorList {
```

Comment:
Has this been migrated to ndbserver_webhook.go, as a part of another PR before?

Reply:
We are not introducing webhooks for NDBServer CRD as of right now. The only validation needed is `required` which is the default behavior right now.